script.js

```diff
@@ -0,0 +1,16 @@
+isThisBoolean(true);  //true
+isThisBoolean(false);  // true
+isThisBoolean('true'); // false
+
+
+function isThisBoolean(providedData) {
+    if ( providedData === false ) {
+        result = true
+    } else {
+        if ( providedData === true ) {
+            result = true
+        } else {
+            result = false}
+    }
+    console.log(result)
+}
```

Comment:
please use [typeof](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof) operator

Reply:
this task is still done in wrong way 
not `dataType` but `cont dataType` + function isThisBoolean shoulr return boolean right now it return undefined